examples/dell5k_sensor_tag_iotcc.py

```diff
@@ -0,0 +1,265 @@
+# The program illustrates the ease of use Liota brings to IoT application developers.
+
+if __name__ == '__main__':
+
+    # create a data center object, IoTCC in this case, using websocket as a transport layer
+    iotcc = IotControlCenter(config['IotCCUID'], config['IotCCPassword'],
+                             WebSocketDccComms(url=config['WebSocketUrl']))
+
+    try:
+        # create a System object encapsulating the particulars of a IoT System
+        # argument is the name of this IoT System
+        edge_system = Dell5KEdgeSystem(config['EdgeSystemName'])
+
+        # resister the IoT System with the IoTCC instance
+        # this call creates a representation (a Resource) in IoTCC for this IoT System with the name given
+        reg_edge_system = iotcc.register(edge_system)
+
+        # these call set properties on the Resource representing the IoT System
+        # properties are a key:value store
+        reg_edge_system.set_properties(config['SystemPropList'])
+
+        # Operational metrics of EdgeSystem
+        cpu_utilization_metric = Metric(
+            name="CPU Utilization",
+            unit=None,
+            interval=10,
+            aggregation_size=2,
+            sampling_function=read_cpu_utilization
+        )
+        reg_cpu_utilization_metric = iotcc.register(cpu_utilization_metric)
+        iotcc.create_relationship(reg_edge_system, reg_cpu_utilization_metric)
+        # call to start collecting values from the device or system and sending to the data center component
+        reg_cpu_utilization_metric.start_collecting()
+
+        cpu_procs_metric = Metric(
+            name="CPU Process",
+            unit=None,
+            interval=6,
+            aggregation_size=8,
+            sampling_function=read_cpu_procs
+        )
+        reg_cpu_procs_metric = iotcc.register(cpu_procs_metric)
+        iotcc.create_relationship(reg_edge_system, reg_cpu_procs_metric)
+        reg_cpu_procs_metric.start_collecting()
+
+        mem_free_metric = Metric(
+            name="Memory Free",
+            unit=None,
+            interval=10,
+            sampling_function=read_mem_free
+        )
+        reg_mem_free_metric = iotcc.register(mem_free_metric)
+        iotcc.create_relationship(reg_edge_system, reg_mem_free_metric)
+        reg_mem_free_metric.start_collecting()
+
+        # Connects to the SensorTag device over BLE
+        sensor_tag_collector = SensorTagCollector(device_name=config['DeviceName'], device_mac=config['DeviceMac'],
+                                                  sampling_interval_sec=5, retry_interval_sec=5,
+                                                  sensors=[Sensors.TEMPERATURE, Sensors.HUMIDITY, Sensors.BAROMETER,
+                                                           Sensors.BATTERY_LEVEL, Sensors.LIGHTMETER,
+                                                           Sensors.ACCELEROMETER, Sensors.GYROSCOPE])
+        sensor_tag = sensor_tag_collector.get_sensor_tag()
```

Comment:
There shouldn't be the requirement of get_sensor_tag(), just have one definition for sensor tag device and use it.